pywt/_cwt.py

```diff
@@ -1,13 +1,40 @@
-import numpy as np
+from math import floor, ceil
 
 from ._extensions._pywt import (DiscreteContinuousWavelet, ContinuousWavelet,
                                 Wavelet, _check_dtype)
 from ._functions import integrate_wavelet, scale2frequency
 
+
 __all__ = ["cwt"]
 
 
-def cwt(data, scales, wavelet, sampling_period=1.):
+import numpy as np
+
+try:
+    # Prefer scipy.fft (new in SciPy 1.4)
+    import scipy.fft
```

Comment:
this also exists in older SciPy version, there's it's an alias of `numpy.fft.fft`. So this should give `AttributeError` (for `next_fast_len`) rather than `ImportError`

Reply:
When I try it in Python 3.7 with Scipy 1.3 I get `ModuleNotFoundError` which is a subclass of `ImportError`.

Reply:
We have not added any test cases incorporating SciPy to the CI runs and should probably do that here as well.

Reply:
ah right yes, `from scipy import fft` is the one that should work. this is fine

Reply:
I think the `AttributeError` case would occur if this had been written:
`from scipy import fft as fftmodule` on old SciPy.  Then when trying to call `fftmodule.fft` an `AttributeError` would occur.